package.json

```diff
@@ -8,8 +8,9 @@
     "url": "git+https://github.com/feedhenry-raincatcher/raincatcher-typescript.git"
   },
   "scripts": {
-    "docs":"typedoc -out ./docs ./packages/",
-    "bootstrap": "lerna bootstrap"
+    "docs": "typedoc -out ./docs ./packages/",
```

Comment:
This commad stopped working for me due to some internal typescript errors. Is that working for you?